CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
         - denoising_tv: The program will make denoising using total variation
         - tomo_ctf_wiener2d_correction: This method performs a CTF correction of a tilt series
         - angular_continuous_assign2_gpu: Make a continuous angular assignment with GPU
+        - continuous_create_residuals: Make better residuals by continuous assignment updates to projections
```

Comment:
Please move it to the next release changes:
Release 3.24.X -

Reply:
done